library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +35,22 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    public fun Context.createShortcut() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            getSystemService(ShortcutManager::class.java)?.let {
+                val shortcut = ShortcutInfo.Builder(this, "id1")
+                    .setShortLabel("Open chucker activity")
+                    .setLongLabel("Open chucker activity")
+                    .setIcon(Icon.createWithResource(this, R.mipmap.chucker_ic_launcher_round))
```

Comment:
Icon is added only as `xxxhdpi` resource. Please migrate to AVD or add other resolutions as well.

Reply:
AVD would be much better. Please remove that added PNG

Reply:
Added other dpi icons

Reply:
Raster icons are OK with me. I can make a follow–up PR with an AVD later on unless you want to change it. :)

Reply:
Ok, let's do that switch to AVD in a follow-up PR ourselves.